tests/Platforms/AbstractPlatformTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Platforms;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\Keywords\KeywordList;
+use Doctrine\DBAL\Schema\Index;
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+
+final class AbstractPlatformTest extends TestCase
+{
+    public function testItThrowsWhenCreatingIndexWithoutColumns(): void
+    {
+        $platform = $this->getMockForAbstractClass(
+            AbstractPlatform::class,
+            [],
+            '',
+            true,
+            true,
+            true,
+            ['createReservedKeywordsList'],
+        );
+        $platform->method('createReservedKeywordsList')
+            ->willReturn($this->createStub(KeywordList::class));
```

Comment:
Let's not do this. I'm perfectly fine with not having this change tested rather than testing it this way.

Reply:
Yeah it's ugly, couldn't find a better way. I'll drop that test.